src/Models_Functions/QSM/getLink_qsm.m

```diff
@@ -1,4 +1,4 @@
-function  link = getLink
+function  link = getLink_qsm
```

Comment:
Instead of it's own function, why don't we juste make it a method of the function "AbstractClass" called getLink(), and override it in the model files? That way every model would just need "model.getLink()" to be called, instead of having a potentially messy way of naming each of the files/functions (and causing mistakes like what is being fixed here).